src/docs/guides/style-guide.md

```diff
@@ -213,7 +213,7 @@ export class Something {
   componentWillUpdate() {}
   componentDidUpdate() {}
   componentWillRender() {}
-  componentShouldRender(newVal: any, oldVal: any, propName: string) {}
+  componentShouldUpdate(newVal: any, oldVal: any, propName: string) {}
```

Comment:
Thanks! Since the style guide states (in the JSDoc above this section):

>   * Ordered by their natural call order, for example
>   * WillLoad should go before DidLoad.

Would you mind moving this line to come before `componentWillUpdate() {}`?